src/tests/unit/endpoint_standard/test_endpoint_standard_enriched_events_facet.py

```diff
@@ -0,0 +1,201 @@
+"""Testing EnrichedEventFacet objects of cbc_sdk.endpoint_standard"""
+
+import pytest
+import logging
+from cbc_sdk.endpoint_standard import EnrichedEventFacet, EnrichedEventFacetQuery
+from cbc_sdk.rest_api import CBCloudAPI
+from cbc_sdk.errors import ObjectNotFoundError, ApiError
+from tests.unit.fixtures.CBCSDKMock import CBCSDKMock
+from tests.unit.fixtures.endpoint_standard.mock_enriched_events_facet import (POST_ENRICHED_EVENTS_FACET_SEARCH_JOB_RESP,
+                                                             GET_ENRICHED_EVENTS_FACET_SEARCH_JOB_RESULTS_RESP_1,
+                                                             GET_ENRICHED_EVENTS_FACET_SEARCH_JOB_RESULTS_RESP_2)
+
+
+log = logging.basicConfig(format='%(asctime)s %(levelname)s:%(message)s', level=logging.DEBUG, filename='log.txt')
+
+
+@pytest.fixture(scope="function")
+def cb():
+    """Create CBCloudAPI singleton"""
+    return CBCloudAPI(url="https://example.com",
+                      org_key="test",
+                      token="abcd/1234",
+                      ssl_verify=False)
+
+
+@pytest.fixture(scope="function")
+def cbcsdk_mock(monkeypatch, cb):
+    """Mocks CBC SDK for unit tests"""
+    return CBCSDKMock(monkeypatch, cb)
+
+
+# ==================================== UNIT TESTS BELOW ====================================
+
+def test_enriched_event_facet_select_where(cbcsdk_mock):
+    """Testing EnrichedEvent Querying with select()"""
+    cbcsdk_mock.mock_request("POST", "/api/investigate/v2/orgs/test/enriched_events/facet_jobs", POST_ENRICHED_EVENTS_FACET_SEARCH_JOB_RESP)
+    cbcsdk_mock.mock_request("GET", "/api/investigate/v2/orgs/test/enriched_events/facet_jobs/08ffa932-b633-4107-ba56-8741e929e48b/results", GET_ENRICHED_EVENTS_FACET_SEARCH_JOB_RESULTS_RESP_1)
+
+    api = cbcsdk_mock.api
+    events = api.select(EnrichedEventFacet).where(process_name="chrome.exe").add_facet_field("process_name")
+    for event in events:
+        assert event.terms is not None
+        assert event.ranges is not None
```

Comment:
This needs to a more detailed assert than not None as event.ranges was not request only facet field. This should be verified as an empty array and the terms should only contain the process_name term

Reply:
Please apply this to all the tests added in this PR

Reply:
Done.